### DISCUSSION TITLE
(Rate) Is el-rate bugged?

### DISCUSSION THREAD

**Discussion by alvinindra** [element-plus/element-plus#4441] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/30944414/142346997-0ea3952e-005a-45f9-82c9-8f1efe0e0c4c.png)

When I use it on my project the star icon is gone too

Does anyone have a solution?

Element-plus version 1.2.0-beta.3

**Comment by emojiiii** [comment]
please make sure your project import  `@element-plus/icons` and registry them

**Comment by sxzz** [comment, marked as answer]
It's a bug.

**Reply by sxzz** [reply]
https://github.com/element-plus/element-plus/issues/4450
